nss-day-cohort-44/rare-rest-api-malicous-vortices
# Custom Property

# ISSUE TICKET 
## Description
A custom property is now attached to the comment model to indicate whether or not the logged in user is the author of the comment. 

## Type of change

# Testing Instructions
1. `git fetch --all `
1. `git checkout wb-custom-property` 
1. GET all comments [http://localhost:8000/comments](url)
1. Check that the property `my_comment` is in each object
1. GET an individual comment [http://localhost:8000/comments/1](url)
1. Check that the property `my_comment` is in each object
1. Keep this branch running to test the client side
